extensions/sql-migration/src/api/utils.ts

```diff
@@ -875,7 +875,7 @@ export async function createManualIRconfigContentContainer(view: azdata.ModelVie
 	if (service) {
 		const account = migrationStateModel._azureAccount;
 		const subscription = migrationStateModel._sqlMigrationServiceSubscription;
-		const resourceGroup = migrationStateModel._sqlMigrationServiceResourceGroup.name;
+		const resourceGroup = service.properties.resourceGroup;
 		const location = service.location;
 		const serviceName = service.name;
```

Comment:
Service name also should be newer one, right? but if it is working fine locally that we can go ahead.

Reply:
service name is the name property of the service and this service is assigned in the previous stage with the newly created service.